tools/Vis_PV.py

```diff
@@ -0,0 +1,286 @@
+    renderView1.CameraFocalPoint = [43.11431452669897, 23.554652286151217, -46.78071794111179]
+    renderView1.CameraViewUp = [0.09790398117889891, -0.5275190119519493, 0.843882990999677]
+    renderView1.CameraParallelScale = 715.9367631158923
+    renderView1.Background = [0.0, 0.0, 0.0]
+    return renderView1
+
+def load_InputData():
+    '''
+    Read data from yaml input file
+    '''
+    Input_stream = open("InputVis_PV.yaml", 'r')
+    Input_data = yaml.load(Input_stream)
+    return Input_data
+
+def readData_Xdmf(xdmf_FilePath): 
+    '''
+    Read data from XDMF file specified in 
+    yaml file.
+    '''
+    try:
+        xDMFReader1 = XDMFReader(FileNames=xdmf_FilePath)
+    except FileNotFoundError:
+        sys.exit("No such file at file location specified")
+    xDMFReader1 = GetActiveSource()
+    renderView1 = GetActiveViewOrCreate('RenderView')
+    xDMFReader1_Display = GetDisplayProperties(xDMFReader1, view=renderView1)
+    return renderView1, xDMFReader1, xDMFReader1_Display
+
+def set_Representation(representation, renderView1, xDMFReader1_Display):
+    '''
+    Set representation and update view 
+    '''
+    xDMFReader1_Display.Representation = representation
+    renderView1.Update()
+    return renderView1, xDMFReader1_Display
+
+def set_colorMap(var, renderView1, xDMFReader1_Display, color_map):
+    '''
+    Set color map to one of preset ParaView color maps.
+    If color map entered is invalid, color map is set to 
+    default.
+    '''
+    ColorBy(xDMFReader1_Display, ('POINTS', var))
+    Var_LUT = GetColorTransferFunction(var)
+    try:
+        Var_LUT.ApplyPreset(color_map, True)
+    except:
+        print('Using default color map')
+        Var_LUT.ApplyPreset('Viridis (matplotlib)', True)
+    xDMFReader1_Display.SetScalarBarVisibility(renderView1, True)
+    xDMFReader1_Display.SetScaleArray = ['POINTS', var]
+    xDMFReader1_Display.ScaleTransferFunction = 'PiecewiseFunction'
+    return xDMFReader1_Display, renderView1, Var_LUT 
+
+def tetrahedralize(xDMFReader1, renderView1):
+    '''
+    Apply tetrahedralize filter and update view
+    '''
+    tetrahedralize1 = Tetrahedralize(Input=xDMFReader1)
+    Hide(xDMFReader1, renderView1)
+    xDMFReader1_Display = Show(tetrahedralize1, renderView1)
+    renderView1.Update()
+    return renderView1, xDMFReader1_Display
+
+def set_Opacity(var, fn_type, opacity_val, renderView1, xDMFReader1, Var_LUT):
+    '''
+    Set opacity based on option chosen in yaml input file:
+    1 : Set to constant opacity specified by user
+    2 : vary opacity as a function of variable (the variable 
+        specified in yaml input file as Variable_name
+    '''
+    if fn_type!=1 and fn_type!=2:
+        sys.exit("The Opacity  option specified in the input file is invalid. Enter 1 or 2 for type of function")
+    #Get range of var array
+    var_range = xDMFReader1.PointData.GetArray(var).GetRange()
+    max = var_range[0]
+    min = var_range[1]
+    N_points = 10
```

Comment:
Could you add a comment why 10 points?

`N_points` -> `num_points` or `num_pts`